mod/parse_url.php

```diff
@@ -92,6 +92,11 @@ function parse_url_content(App $a)
 		}
 	}
 
+	if (!empty($_GET['isComment'])) {
+		echo $br . '[url]' . $url . '[/url]';
```

Comment:
Please change this to `[url=<Page URL>]<Page Title>[/url]` if there's a non empty title in the response.